### PR TITLE
Use Jupyter Releaser

### PR DESCRIPTION
**Pull request by afshin**
Use [jupyter_releaser](https://github.com/jupyter-server/jupyter_releaser) to make the release process more standardized and simple
Addresses https://github.com/jupyter-server/jupyter_releaser/issues/15

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+### Install release dependencies
 
 ```bash
-conda install -c conda-forge nodejs babel twine
+conda install -c conda-forge nodejs babel twine build
 npm install -g po2json
 ```
 
-## Update the version
+### Update the version
```
Comment (kevin-bates) [inline RELEASE.md:45]: Should this include `vim pyproject.toml`?  (Same for the 'Change back to dev version' below)

File: notebook/_version.py [hunk 2]
```diff
@@ -2,12 +2,19 @@
 store the current version info of the notebook.
 
 """
+import re
+
+# Version string must appear intact for tbump versioning
+__version__ = '6.5.0.dev0'
```
Comment (kevin-bates) [inline notebook/_version.py:8]: Should this be `'6.4.0a1'`?

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+[tool.jupyter-releaser.hooks]
+after-bump-version = "python setup.py jsversion"
+before-build-python = ["pip install babel", "npm install -g po2json"]
+
+[tool.jupyter-releaser.options]
+ignore-glob = ["docs/source/examples/Notebook/Working With Markdown Cells.ipynb", "docs-translations/**/README.md", "docs/source/contributing.rst", "docs/source/examples/Notebook/JavaScript Notebook Extensions.ipynb", "CHANGELOG.md"]
+
+[tool.tbump.version]
+current = "6.5.0.dev0"
```
Comment (kevin-bates) [inline pyproject.toml:17]: Should this be `"6.4.0a1"`?

**Review by kevin-bates — COMMENTED** [review]
Thanks Darian - this looks helpful.

File: notebook/_version.py (hunk 2 above)
Comment (kevin-bates) [inline notebook/_version.py:8]:
```suggestion
__version__ = '6.4.0a1'
```

File: RELEASE.md (hunk 1 above)
Comment (afshin) [reply RELEASE.md:45]: No, the version source of truth is in `_version.py` and is read in by `setup.cfg` by value.

File: pyproject.toml (hunk 3 above)
Comment (kevin-bates) [inline pyproject.toml:17]:
```suggestion
current = "6.4.0a1"
```

**Pull request merged**